### PR TITLE
Update FAQ

### PR DESCRIPTION
**Pull request by ocefpaf**
Also add two new questions about git and engagement.

This is based on #67. We should merge that one first!

### CODE REVIEW

**Comment by emiliom** [comment]
Why are the two files other than 01-applicants.md, from my PR #67, also here?

**Comment by ocefpaf** [comment]
> Why are the two files other than 01-applicants.md, from my PR #67, also here?

Like I said above. I based on you commits. See the commit history. It is easier than dealing with conflicts later. Let me rebase with `source`, now that your PR was merge, and they should "disappear" from the diff with `source`.

File: _pages/01-applicants.md [hunk 1]
```diff
@@ -25,51 +25,90 @@ For additional questions, please email <a href="mailto:oceanhkw@uw.edu" target="
...
+
+Yes, we welcome applications from previous participants,
+but priority will be given to new applicants.
+If you are interested in being a TA (teaching assistant) during the hackweek,
+please mention that in the application.
+
+**Q: Am I supposed to participate in all tutorials and presentations?**
 
-Yes, we welcome applications from previous participants, but priority will be given to new applicants. If you are interested in being a TA (teaching assistant) during the hackweek, please mention that in the application.
+Yes. While we understand that not all topics may be 100% relevant to you we want to maximize engagement. Note that we post the lessons and videos online later, so if you are interested in only a portion of the OHW consider that an option rather than taking someone spot.
```
Comment (ocefpaf) [inline _pages/01-applicants.md:112]: I would like some help making this one more "positive."

File: _pages/01-applicants.md [hunk 2]
```diff
@@ -25,51 +25,90 @@ For additional questions, please email <a href="mailto:oceanhkw@uw.edu" target="
...
-about its impact on plans for a virtual event in 2020 and in-person events in 2021. 
-Please check this web site regularly for updates.
-To stay up to date on the current impact of the coronavirus emergency and government recommendations,
-see the US Federal web site <a href="https://www.coronavirus.gov">https://www.coronavirus.gov</a> 
-and your state and local positions.
+We welcome participants from all career stages and encourage applications from graduate students,
+postdocs and early career researchers.
 
-**Q: I am an undergrad / first year grad / faculty / etc. Is this program right for me?**
+Before applying consider what you can bring to the even:
```
Comment (emiliom) [inline _pages/01-applicants.md:33]: "event" is mispelled

File: _pages/01-applicants.md [hunk 3]
```diff
@@ -25,51 +25,90 @@ For additional questions, please email <a href="mailto:oceanhkw@uw.edu" target="
...
 
-We want to build an inclusive Oceanography community regardless of people’s language of choice and programming level. Since the event is only one week long, the tutorials will be presented in one language (Python), and participants will benefit the most if they have basic familiarity with this language. We expect that you have gone through the [Software Carpentry Python tutorial](http://swcarpentry.github.io/python-novice-gapminder) to get familiar with the syntax in advance of the program. The more experience you can obtain before the beginning of the program, the easier it will be to follow the tutorials and contribute to the projects.
+We want to build an inclusive Oceanography community regardless of people’s language of choice and programming level.
+Note that most of the tutorials will be presented in Python,
+and participants will benefit the most if they have basic Python skills.
+
+We expect that you have gone through the [Software Carpentry Python tutorial](http://swcarpentry.github.io/python-novice-gapminder) to get familiar with the syntax in advance of the program.
```
Comment (emiliom) [inline _pages/01-applicants.md:51]: For consistency I think we should copy and paste here (replace these two sentences) the relevant text from the application form, in the description of the Programming Skills section.

File: _pages/01-applicants.md [hunk 4]
```diff
@@ -25,51 +25,90 @@ For additional questions, please email <a href="mailto:oceanhkw@uw.edu" target="
...
 **Q: I have strong computational skills but have not worked in oceanography?**
 
-To ensure that participants will be interested in the oceanography problems they are solving and can give back to the field later on, we require that participants have had at least some familiarity or prior experience with oceanography data.
+To ensure that participants will be interested in the oceanography problems they are solving and can give back to the field later on,
+we require that participants have had at least some familiarity or prior experience with oceanography data.
 
 **Q: I work in an ocean engineering/robotics/consulting company. Can I apply?**
 
-Make sure to explain your motivation for participation in the application form! We may hold a career panel discussion, so please mention if you are interested in participating.
+We expect participants from the private sector to pay for their own expenses.
```
Comment (emiliom) [inline _pages/01-applicants.md:94]: There are no direct costs involved in this virtual event. This sentence was from last year. Please remove.

File: _pages/01-applicants.md (hunk 1 above)
Comment (emiliom) [reply _pages/01-applicants.md:112]: I've reviewed everything else but will leave this one to @leewujung !

**Comment by emiliom** [comment]
I reviewed everything except the last question. Looks great! See my small inline comments.

**Pull request merged**